language-extensions/owl-extension/src/main/java/dev/ikm/tinkar/ext/lang/owl/Rf2OwlToLogicAxiomTransformer.java

```diff
@@ -125,6 +127,7 @@ protected Void compute() throws Exception {
         updateMessage("Completed transformation");
         LOG.info("Completed processing of {} stated OWL expressions...", count.get());
 
+        EntityService.get().endLoadPhase();
```

Comment:
If the begin/endLoadPhase is necessary here, probably bettern to use a try/finally pattern to make sure there's no way to exit this method without the call to endLoadPhase